src/plugins_manager.coffee

```diff
@@ -0,0 +1,56 @@
+define [
+  'plugins_settings',
+  'jsonp'
+], (PluginsSettings, JSONP) ->
+
+  class PluginsManager
+    constructor: (@session) ->
+      @loaded_plugins = [] # use it in order to load a plugin only once
+      @enabled_plugins_fetched = @_fetchEnabledPlugins().then (response) => @available_plugins = response.plugins
+
+    notify: (action, data) ->
+      @enabled_plugins_fetched.then =>
```

Comment:
Why don't you re-use `@_fetchEnabledPlugins().then` instead of `@enabled_plugins_fetched`?